src/mol-canvas3d/controls/trackball.ts

```diff
@@ -40,8 +41,16 @@ export const TrackballControlsParams = {
     zoomSpeed: PD.Numeric(7.0, { min: 1, max: 15, step: 1 }),
     panSpeed: PD.Numeric(1.0, { min: 0.1, max: 5, step: 0.1 }),
 
-    spin: PD.Boolean(false, { description: 'Spin the 3D scene around the x-axis in view space' }),
-    spinSpeed: PD.Numeric(1, { min: -20, max: 20, step: 1 }),
+    animate: PD.MappedStatic('off', {
```

Comment:
Use ``cycle`` styles for this?

Reply:
nah, hinders discoverability and is an odd ux pattern here, imho